deposit_contract.sol

```diff
@@ -80,8 +79,10 @@ contract DepositContract is IDepositContract {
         require(deposit_count < MAX_DEPOSIT_COUNT);
 
         // Check deposit amount
+        require(msg.value >= 1 ether);
+        require(msg.value % GWEI == 0, "DepositContract: deposit value not a multiple of gwei");
```

Comment:
Can we keep all the revert strings in the #11 PR?

Also in this one the first `require` is missing the string.

Reply:
well, we need to decide on an ordering of these PRs first. This one assumes `#11` is merged first. But you are right that the above one is inconsistent with that assumption